collectors/ioping.plugin/metadata.yaml

```diff
@@ -33,22 +33,44 @@ overview:
       description: ''
 setup:
   prerequisites:
-    list: []
+    list: |
+      It is necessary to have the command `ioping` installed on your host. You can install the command by passing the argument `install` to the plugin:
+
+      ```bash
+      /usr/libexec/netdata/plugins.d/ioping.plugin install
+      ```
   configuration:
     file:
-      name: ''
-      description: ''
+      name: 'ioping.conf'
+      description: 'File with options to specify hardware to monitor and arguments for ioping command.'
     options:
       description: ''
       folding:
-        title: ''
+        title: 'Config options'
         enabled: true
-      list: []
+      list:
+        - name: update_every
+          description: Data collection frequency.
+          default_value: 1s
+          required: false
+        - name: destination
+          description: The directory/file/device to ioping.
+          default_value: ""
+          required: true
+        - name: request_size
+          description: The request size in bytes to ioping the destination (symbolic modifiers are supported)
+          default_value: 4k
+          required: false
+        - name: ioping_opts
+          description: Options passed to `ioping` commands.
+          default_value: -T 1000000
+          required: false
     examples:
       folding:
         enabled: true
         title: ''
-      list: []
+      list: |
```

Comment:
i am not sure what this is - if you want to do an example i think needs to be something like this: https://github.com/netdata/netdata/blob/master/collectors/python.d.plugin/example/metadata.yaml#L98